class-tgm-plugin-activation.php

```diff
@@ -1258,7 +1274,8 @@ public function register( $plugin ) {
 		 *
 		 * @param string $source The source of the plugin as provided, either empty (= WP repo), a file path
 		 *                       (= bundled) or an external URL.
-		 * @return string        'repo', 'external', or 'bundled'
+		 *
+		 * @return string 'repo', 'external', or 'bundled'
```

Comment:
I prefer space between `@param` and `@return`, like this, but unfortunately the WP handbook says not to.